src/Multi_persistence/include/gudhi/Line.h

```diff
@@ -95,7 +98,7 @@ template <typename T> class Line {
 };
 template <typename T> inline bool Line<T>::check_direction() const {
   bool is_trivial=true;
-  for (const auto& stuff : basepoint_){
+  for (const auto& stuff : basepOne_critical_filtrationoint_){
```

Comment:
A wrong paste?

Reply:
Thanks ! I have noticed that my IDE (Visual Studio Code) was copying things randomly (maybe when scrolling files, I may click). I changed my IDE. To be continued...
Done on [a42608e](https://github.com/GUDHI/gudhi-devel/pull/1123/commits/a42608ece812a62c49dbb145fbfef82cd30d9129)